index.html

```diff
@@ -18,8 +18,8 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/angularjs-slider/2.13.0/rzslider.min.js"></script>
     <script src="./assets/lib/proj4js/proj4.js"></script>
     <script src="./config/3857.js"></script>
-    <!-- <script src="./assets/lib/ol3/ol-debug.js"></script> -->
-    <script src="./assets/lib/ol3/ol.js"></script>
+    <script src="./assets/lib/ol3/ol-debug.js"></script>
```

Comment:
when using ol-debug, wouldn't it be nicer to have an index-dev.html?
Or was this just for testing purposes and forgotten?